### DISCUSSION TITLE
You impacted the market! Values raised/lowered

### DISCUSSION THREAD

**Discussion by jmmullins** [jeremyckahn/farmhand#438] [Q&A] [answered]
I've noticed this a few times now, but I'm hesitant to just report it as a bug without knowing more about how the system was designed to work.. 

Periodically, I'll see the notifications for impacting the market with whatever crops it says the value was raised/lowered on, but I've noticed that these don't always seem to be showing up at the same time as a change in market value, particularly as it pertains to surge/crash prices.

Do the surge/crash events override what the typical sale value would be? e.g. Is the value of said crop (let's pick carrots for this example) still fluctuating in the background for players in the global room but the surge/crash is only happening to/for me for the few "days" until things normalize? Or would carrots be at surge/crash prices for everyone in the global room?

It's been easier to notice this when at a crash price because I'm watching for it to recover every day, but it's an odd user experience to see "value raised: carrots" when you're staring at your inventory where it says carrots are still $12.50 for the third day in a row. 

I've also seen the same though when something is at surge price; subsequent value lowered notification yet the surge price is still in effect, so it's confusing. 

If the surge/crash only applies to me, that could account for why notifications are coming through for affecting the market even though the market hasn't changed, but again.. I'm not sure how the logic is currently structured around this.

Final note: There's a chance that this is NOT specific to times with surge/crash prices, I haven't watched the values _that_ closely to know if it does or doesn't happen on "normal" days

UPDATE: I've now seen this from the perspective of 2 players playing simultaneously, and it does seem like surge/crash events are not shared across the other players in the online room, but this is definitely unclear; I'd been assuming they happened to everyone in the room.

**Comment by jeremyckahn** [comment, marked as answer]
You've pretty much got it; price surges/crashes are local to only the player that sees them and not the room's shared market. I think that's the right design choice because a local surge or crash will cause a player to make purchases and sales that go against what the shared market prices would normally incentivize. This behavior tends to introduce more variability into the market. That said, you raise a good point about questionable messaging about how the player has impacted the market locally and globally. Maybe changing the notification copy from "You impacted the market!" to something like "You impacted the **room** market!" might help here?
